code/src/nuvla/job/actions/application_docker_compose_validate.py

```diff
@@ -19,15 +19,30 @@ def __init__(self, _, job):
 
     @staticmethod
     def get_env_to_mute_undefined(module_content):
```

Comment:
Could you add some unit-test for that function. And an entry in the changelog.md file for that change.